guardian/grid
# remove google analytics code from Kahuna

## What does this change?

Removes the code for Google Analytics:
- googleTrackingId property from config
- the GA script tags in main.scala.html
- the analytics module in the angular app

The account it used was turned off in July and the data wasn't being used before then,  see https://trello.com/c/MvsOLUwA/1435-remove-ga-from-grid.


## How should a reviewer test this change?

Runs locally without any noticable change (there was no googleTrackingId in the DEV config, so the GA scripts were not rendered, but the the line in the config script (visible in view-source:https://media.local.dev-gutools.co.uk/) will be gone.

When deployed to CODE, the should work as before, but with no googleTrackingId loaded into config and no https://www.google-analytics.com/analytics.js script

## How can success be measured?

Still works as before, less redundant code.

## Who should look at this?


## Tested? Documented?
